commons-akka/src/main/scala/com/avsystem/commons/rpc/akka/serialization/ByteStringLinearOutput.scala

```diff
@@ -9,7 +9,7 @@ import com.avsystem.commons.serialization.{ListOutput, ObjectOutput, Output}
 /**
   * @author Wojciech Milewski
   */
-private[akka] class ByteStringLinearOutput(builder: ByteStringBuilder) extends Output {
+final class ByteStringLinearOutput(private val builder: ByteStringBuilder) extends AnyVal with Output {
```

Comment:
We already have `StreamOutput` and `StreamInput` which can be also used for writing `ByteString`s (as in `ByteStringSerialization` for Redis driver). We might want to get rid of this implementation.